docs/labs/6-Lists.md

```diff
@@ -104,6 +104,8 @@ void printList(ArrayList<String> list) {
 }
 ```
 
+Here is a [comparison between Arrays and ArrayLists](https://www.w3schools.blog/difference-between-array-and-arraylist-in-java).
```

Comment:
can you elaborate with another sentence or revise the current one for why we would want this, or like "for a more in-depth thing" or some reason why they would want this (like are you envisioning that they are still confused?  etc)